tests/worksheets/src/test/scala/tests/worksheets/WorksheetSuite.scala

```diff
@@ -285,12 +318,8 @@ class WorksheetSuite extends BaseSuite {
     """|import $dep.`org.json4s:json4s-native_2.13:3.6.9`
        |import org.json4s._
        |import org.json4s.native.JsonMethods._
-       |<parse("{ \"numbers\" : [1, 2, 3, 4] }")> // : JValue = JObject(o...
-       |res0: JValue = JObject(
-       |  obj = List(
-       |    (
-       |      "numbers",
-       |...
+       |<parse("{ \"numbers\" : [1, 2, 3, 4] }")> // : JValue = JObject(L...
+       |res0: JValue = JObject(List((numbers,JArray(List(JInt(1), JInt(2), JInt(3), JInt(4))))))
        |""".stripMargin
   )
```

Comment:
we can add test importing scalatags to confirm all this embedding pprint worked

Reply:
Changed the test to use scalatags. Good idea!